microsoft-edge/webview2/concepts/versioning.md

```diff
@@ -27,29 +24,44 @@ The prerelease package is a superset of the release package with the additional
 *   Experimental APIs:  For more information, navigate to the [Experimental APIs](#experimental-apis) section.  
 
-## Experimental APIs  
+### Experimental APIs  
 
 The WebView team is testing experimental APIs that may be included in future releases.  The experimental APIs are marked as `experimental` in the SDK.  Experimental APIs may ship as fully stable APIs in the release package.  You can evaluate the Experimental APIs and share feedback using the [WebView feedback repo][GithubMicrosoftedgeWebviewfeedback].  
 
+
+
 > [!CAUTION]
-> Avoid using the experimental APIs in production apps.  
+> Experimental APIs may be introduced, modified, and removed from SDK to SDK.  Experimental APIs may not be available in your installed version of the WebView2 Runtime.  Avoid using the experimental APIs in production apps.  
+
+### Roadmap  
+
+The release package contains all of the stable, supported Win32 C/C++ APIs.  In the future, the release package will contain all stable, supported .NET APIs when they are made generally available.  The prerelease package contains experimental APIs that are subject to change based upon your feedback and shared insights. 
 
 ## Matching WebView2 Runtime versions  
+To develop a WebView2 application, you must install either the [WebView2 Runtime][MicrosoftDeveloperEdgeWebview2] or a [non-stable Microsoft Edge channel][MicrosoftedgeinsiderDownload]. There are two things to take into account when determining the minimum version of the [WebView2 Runtime][MicrosoftDeveloperEdgeWebview2] required.
+
+1. **The minimum required version of the SDK:** This is specified in the WebView2 [Release Notes][Releasenotes]. For example, for SDK version [1.0.622.22](https://docs.microsoft.com/microsoft-edge/webview2/releasenotes#1062222), you must install either the [WebView2 Runtime][MicrosoftDeveloperEdgeWebview2] or a [non-stable Microsoft Edge channel][MicrosoftedgeinsiderDownload] with a build number of **86.0.616.0** or later. The minimum version required will only change when there is a breaking change in the web platform.
+
+2. **The minimum required version for the interfaces and APIs you depend on:** New interfaces and APIs are added periodically to WebView2. Different APIs and interfaces within an SDK require different versions of the WebView2 Runtime depending on when they were introduced. For example, an API or interface added in SDK version [1.0.622.22](https://docs.microsoft.com/microsoft-edge/webview2/releasenotes#1062222) will have a minimum WebView2 Runtime requirement of **86.0.616.0**. An API or interface added in a subsequent SDK release will have a minimum WebView2 Runtime requirement that corresponds to the version of the Runtime that is released with that version of the SDK. You can find the minimum WebView2 Runtime version required by an interface or API [programmatically](#determining-interface-and-api-WebView2-Runtime requirement).
```

Comment:
> An API or interface added in a subsequent SDK release will have a minimum WebView2 Runtime requirement that corresponds to the version of the Runtime that is released with that version of the SDK. 

Explain how can a developer find the corresponding version of an API.

> You can find the minimum WebView2 Runtime version required by an interface or API [programmatically](#determining-interface-and-api-WebView2-Runtime requirement).

QI/try-catch tells you whether RT has the interfaces you use, it doesn't tell you the min RT required.